src/Microsoft.Tye.Hosting/TyeDashboardApi.cs

```diff
@@ -31,12 +32,15 @@ public TyeDashboardApi()
             };
 
             _options.Converters.Add(new JsonStringEnumConverter(JsonNamingPolicy.CamelCase));
+            _processRunner = processRunner;
         }
 
         public void MapRoutes(IEndpointRouteBuilder endpoints)
         {
             endpoints.MapGet("/api/v1", ServiceIndex);
             endpoints.MapGet("/api/v1/services", Services);
+            endpoints.MapGet("/api/v1/services/{name}/stop", ServiceStop);
```

Comment:
These shouldn't be GET's. Make them posts.

Reply:
done!